tst/EnergyPlus/unit/StandardRatings.unit.cc

```diff
@@ -1284,12 +1291,14 @@ TEST_F(EnergyPlusFixture, ChillerIPLVTestWaterCooledReform)
                     state->dataChillerReformulatedEIR->ElecReformEIRChiller(1).ChillerEIRFTIndex,
                     state->dataChillerReformulatedEIR->ElecReformEIRChiller(1).ChillerEIRFPLRIndex,
                     state->dataChillerReformulatedEIR->ElecReformEIRChiller(1).MinUnloadRat,
-                    IPLV,
+                    IPLVSI,
+                    IPLVIP,
                     state->dataChillerReformulatedEIR->ElecReformEIRChiller(1).CondVolFlowRate,
                     1,
                     1.0);
 
-    EXPECT_DOUBLE_EQ(round(IPLV * 100) / 100, 4.83);
+    EXPECT_DOUBLE_EQ(round(IPLVSI * 100) / 100, 4.83); // 4.83 IPLV.SI (AHRI Std 551/591 Test Condition)
+    EXPECT_DOUBLE_EQ(round(IPLVIP * 100) / 100, 4.95); // 16.89 IPLV.IP (AHRI Std 550/590 Test Condition)
```

Comment:
Same here. The `IPLV.SI` and `IPLV.IP` values side-by-side for the same chiller but different AHRI test conditions.